src/riak_kv_qry_compiler.erl

```diff
@@ -1640,6 +1672,39 @@ basic_select_arith_1_test() ->
        Sel
       ).
 
+varchar_literal_test() ->
+    {ok, Rec} = get_query("SELECT 'hello' from mytab"),
```

Comment:
This is just to minimise test data. I'm not sure about plans to do things like `SELECT 2 FROM dual`, maybe if there is ever a need for sub queries.